prow/cmd/deck/trace.go

```diff
@@ -184,6 +217,12 @@ func validateTraceRequest(r *http.Request) (int, error) {
 		return 0, fmt.Errorf("need either %q, %q, and %q or %q to be specified",
 			github.PrLogField, github.RepoLogField, github.OrgLogField, github.EventGUID)
 	}
+	if ic != "" && eventGUID != "" {
+		return 0, fmt.Errorf("cannot specify both issuecomment (%s) and event-GUID (%s)", ic, eventGUID)
+	}
+	if ic != "" && (pr == "" || repo == "" || org == "") {
+		return 0, fmt.Errorf("cannot specify issuecomment (%s) without org, repo, and pr parameters", ic)
```

Comment:
Why? Aren't they unique across GitHub?

Reply:
Yes, they are. Just verified this with Github folks. Updated.